R/shift_configuration.R

```diff
@@ -27,7 +27,8 @@
 #' \item{Y}{input trait vector/matrix.}
 #' \item{tree}{input tree.}
 #' \item{shift.configuration}{estimated shift positions, i.e. vector of indices of edges where the estimated shifts occur.}
-#' \item{shift.values}{estimates of the shift values.}
```

Comment:
Hello Mohammad, Thanks for changing it. I have also changed it in my fork of `l1ou` as well.